ryml build failure

**Host Environment**
- OS: macOS
- Compiler: clang

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install xxxx`

**Failure logs** 

```
Detecting compiler hash for triplet x64-osx...
The following packages will be built and installed:
  * c4core[core]:x64-osx -> 2021-06-26
    ryml[core]:x64-osx -> 2021-06-29
Additional packages (*) will be modified to complete this operation.
Could not locate cached archive: /Users/draveness/.cache/vcpkg/archives/a2/a2af6a3d53e60cc9f945d8cf5d5a70b01ec019ba4125573b2ce9d006506c561a.zip
Could not locate cached archive: /Users/draveness/.cache/vcpkg/archives/a5/a5fa2b92e66756f80694ca7f34ab34dbcfd4743808c207a01fde33ec8e06d191.zip
Starting package 1/2: c4core:x64-osx
Building package c4core[core]:x64-osx...
CMake Error at scripts/cmake/vcpkg_fail_port_install.cmake:96 (message):
  Target 'OSX' not supported by c4core!

Call Stack (most recent call first):
  ports/c4core/portfile.cmake:3 (vcpkg_fail_port_install)
  scripts/ports.cmake:141 (include)


Error: Building package c4core:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: c4core:x64-osx
  Vcpkg version: 2021-07-26-unknownhash

Additionally, attach any relevant sections from the log files above.
```


**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

--- a/ports/c4core/portfile.cmake
+++ b/ports/c4core/portfile.cmake
@@ -1,9 +1,5 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
-vcpkg_fail_port_install(
-    ON_TARGET "OSX"
-)
-
 # Get c4core src
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH

--- a/ports/c4core/vcpkg.json
+++ b/ports/c4core/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "c4core",
   "version-date": "2021-07-18",
+  "port-version": 1,
   "description": "Library of low-level C++ utilities",
-  "homepage": "https://github.com/biojppm/c4core",
-  "supports": "!osx"
+  "homepage": "https://github.com/biojppm/c4core"
 }

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1678,7 +1678,6 @@ zkpp:x64-windows-static-md=fail
 zkpp:arm64-windows=fail
 zkpp:x64-uwp=fail
 zkpp:arm-uwp=fail
-c4core:x64-osx=fail
 
 # Official downloading server of CTP library is only guaranteed to be available during trading hours of China futures market
 # Skip CI to avoid random failures

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1106,7 +1106,7 @@
     },
     "c4core": {
       "baseline": "2021-07-18",
-      "port-version": 0
+      "port-version": 1
     },
     "caf": {
       "baseline": "0.17.6",

--- a/versions/c-/c4core.json
+++ b/versions/c-/c4core.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c1ef064ec4033e2fca5504b4eb44338453050f7f",
+      "version-date": "2021-07-18",
+      "port-version": 1
+    },
+    {
       "git-tree": "eae79eece3a7383d94cc686275624cb49ee6a68e",
       "version-date": "2021-07-18",
       "port-version": 0

